labs/lab-10/tasks/overflow-for-binary/tests/test_overflow_for_binary.c

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+
+#include "graded_test.h"
+
+char *correct_payload = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPR0N";
+
+static int test_payload(void)
+{
+	char payload[1000];
+
+	printf("Spune payload-ul: ");
```

Comment:
Use English, not Romanian, like the rest of the course. This whole file will be changed, but keep this in mind.